modules/cluster-logging-visualizer-indices.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-visualizer.adoc
+
+[id="cluster-logging-visualizer-indices_{context}"]
+= Defining Kibana index patterns
+
+An index pattern defines the Elasticsearch indices that you want to visualize. To explore and visualize data in Kibana, you must create an index pattern. 
+
+.Prerequisites
+
+* A user must be able to view the pods and logs in the `default` project in order to list the *infra* and *audit* indices in Kibana. 
++
+For example:
++
+[source,terminal]
+----
+$ oc auth can-i get pods/logs -n default
+----
++
+.Example output
+[source,terminal]
+----
+yes
+----
+
+* Elasticsearch documents must indexed before you can create index patterns. This is done automatically, but might take a few minutes in a new or updated cluster.
+
+.Procedure
+
+To define index patterns and create visualizations:
+
+. Launch Kibana by clicking the Application Launcher {launch} and select *Logging*.
+
+. Create your Kibana index patterns:
+
+** Regular users must manually create index patterns to see logs for their projects. Users should create a new index pattern named `app` and use the `@timestamp` time field to view their container logs. 
+
+** Admin users need to create index patterns using the *app*, *infra*, and *audit* prefix with the `@timestamp` time field. 
```

Comment:
the index pattern should be just `app` or `infra` or `audit` using "prefix" may be misleading and users may then do `app-*`